src/lib.rs

```diff
@@ -1,9 +1,10 @@
 pub mod raw;
 
-mod environment;
+pub mod environment;
 pub use environment::Environment;
```

Comment:
Good catch! I had an error in my reasoning, than I made that module public. I think we should keep only the reexport for now